docker/rootfs/etc/s6-overlay/s6-rc.d/log-prepare/up

```diff
@@ -0,0 +1 @@
+/etc/s6-overlay/s6-rc.d/log-prepare/run
```

Comment:
Would you mind explaining briefly what these different files are used for / pointing to? Or is there a set of docs that explain this new format?

Reply:
Take a look here: https://github.com/just-containers/s6-overlay#logging

It's almost comprehensive.

Reply:
TL;DR:

https://github.com/just-containers/s6-overlay#logging:~:text=That%27s%20a%20lot%20of%20files!%20A%20summary%20of%20what%20it%20all%20means%20is%3A